### PR TITLE
N21: Add discussion links

### PR DESCRIPTION
**Pull request by 17cupsofcoffee**
_Part of #589_

### CODE REVIEW

File: content/news/021/index.md [hunk 1]
```diff
@@ -1018,13 +1018,11 @@ Want something mentioned in the next newsletter?
 Also, subscribe to [@rust_gamedev on Twitter][@rust_gamedev]
 or [/r/rust_gamedev subreddit][/r/rust_gamedev] if you want to receive fresh news!
 
-<!--
-TODO: Add real links and un-comment once this post is published
 **Discuss this post on**:
-[/r/rust](TODO),
-[Twitter](TODO),
+[/r/rust](https://reddit.com/r/rust/comments/n8g7nc/this_month_in_rust_gamedev_21_april_2021),
```
Comment (ozkriff) [inline content/news/021/index.md:1022]: :thinking:  i guess one reddit link should be enough? /r/rust_gamedev is the main one so I'd remove the link to /r/rust crosspost

Comment (17cupsofcoffee) [reply content/news/021/index.md]: Fixed!

**Comment by 17cupsofcoffee** [comment]
Also added a fix for a broken link, as per https://github.com/rust-gamedev/rust-gamedev.github.io/issues/589#issuecomment-835832291.

**Review by ozkriff — APPROVED** [review]

File: content/news/021/index.md [hunk 2]
```diff
@@ -644,7 +644,7 @@ Rust discovery.
 
 ![How To Write a Crash Reporter](wor-crash-reporter.jpg)
 
-[@masonremaley][mason-remaley] wrote [an article](crash-reporter) walking
+[@masonremaley][mason-remaley] wrote [an article][crash-reporter] walking
```
Comment (ozkriff) [inline content/news/021/index.md:647]: dammit, these typos in md link syntax are so hard to notice :(

Comment (17cupsofcoffee) [reply content/news/021/index.md:647]: I miss them every time 😩

**Pull request merged**